turbo.json

```diff
@@ -4,84 +4,72 @@
         "build": {
             "dependsOn": ["^build"],
             "inputs": [
-                "**/tsconfig.json",
-                "packages/**/*.{js,json,ts,sol}",
-                "!packages/**/{artifacts,cache,dist,typechain-types}/**"
+                "tsconfig.json",
+                "**/build.tsconfig.json",
+                "**/rollup.config.ts",
+                "packages/**/*.{js,json,ts}",
+                "!packages/**/dist/**"
             ],
-            "outputs": ["packages/**/{artifacts,dist,typechain-types}/**"]
+            "outputs": ["packages/**/dist/**"]
         },
-        "//#_build:libraries": {
+        "//#_build": {
             "inputs": [
                 "tsconfig.json",
+                "**/build.tsconfig.json",
+                "**/rollup.config.ts",
                 "packages/**/*.{js,json,ts}",
-                "!packages/**/{artifacts,cache,dist,typechain-types}/**"
+                "!packages/**/dist/**"
             ]
         },
+        "//#_test": {
+            "dependsOn": [
+                "@zk-kit/imt#build",
+                "@zk-kit/lean-imt#build",
+                "@zk-kit/baby-jubjub#build",
+                "@zk-kit/eddsa-poseidon#build",
+                "@zk-kit/lazytower#build",
+                "@zk-kit/poseidon-cipher#build",
+                "@zk-kit/poseidon-proof#build",
+                "@zk-kit/smt#build",
+                "@zk-kit/utils#build"
```

Comment:
@sripwoud Is there a way to avoid listing all packages here? Isn't it possible to put 1 command somehow?

Reply:
Hmm I don't remember why I wrote this but there must have been a reason.

I remember it had to do with dealing with a root task and wanting to specify some workspace tasks as dependencies.
So I ended using this syntax
https://turbo.build/repo/docs/core-concepts/monorepos/task-dependencies#from-arbitrary-workspaces

now we use the root test task that needs everything to be build first.
So did you try simply?

```json
   "//#test":{ "dependsOn": ["build"] ...}
```